docs/source/operations/getting-started.rst

```diff
@@ -0,0 +1,354 @@
+***************
+Getting started
+***************
+
+
+This page gives you directions to locally run the Substra stack. This deployment is made of:
+
+* 1 orchestrator (running in standalone mode, i.e. storing data in its own local database)
+* 2 backends (running in two organisations, ``org-1`` and ``org-2``)
+* 1 frontend
+
+It allows you to run the examples and start using substra SDK (also known as substra).
+
+Prerequisites
+=============
+
+Hardware
+--------
+
+The following table indicates the resources needed to run the Substra stack locally. The minimal profile represents resources needed to be able to run the stack, whereas the recommended profile describes how much is needed to have a comfortable development experience.
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * -
+     - CPU
+     - Hard drive space (Go)
```

Comment:
Nit: the table would be more readable if the units (GB) were in each table cell instead of the header (e.g. 35 GB)